alerting/src/main/kotlin/org/opensearch/alerting/DocumentReturningMonitorRunner.kt

```diff
@@ -67,6 +68,18 @@ object DocumentReturningMonitorRunner : MonitorRunner {
     ): MonitorRunResult<DocumentLevelTriggerRunResult> {
         logger.info("Document-level-monitor is running ...")
         var monitorResult = MonitorRunResult<DocumentLevelTriggerRunResult>(monitor.name, periodStart, periodEnd)
+
+        // TODO: is this needed from Charlie?
+        try {
+            monitorCtx.alertIndices!!.createOrUpdateAlertIndex()
+            monitorCtx.alertIndices!!.createOrUpdateInitialAlertHistoryIndex()
+            monitorCtx.alertIndices!!.createOrUpdateInitialFindingHistoryIndex()
+        } catch (e: Exception) {
+            val id = if (monitor.id.trim().isEmpty()) "_na_" else monitor.id
+            logger.error("Error loading alerts for monitor: $id", e)
+            return monitorResult.copy(error = e)
+        }
+
```

Comment:
can we improve exception handling here instead of just using `java.lang.Exception`?

Reply:
This is currently how alerting works with the existing monitor types, we can take a look at this in a future PR to improve exception handling on all the monitor types